src/objecttypes/conf/dev.py

```diff
@@ -107,8 +107,9 @@
     r"django\.db\.models\.fields",
 )
 
-if "test" in sys.argv:
-    TWO_FACTOR_FORCE_OTP_ADMIN = False
+# None of the authentication backends require two-factor authentication.
+if config("DISABLE_2FA", default=True):  # pragma: no cover
```

Comment:
Could you please add `DISABLE_2FA` to base config in the same way as in Objects API? I think they should be consistent

Reply:
done